packages/image_picker/image_picker/README.md

```diff
@@ -63,13 +63,16 @@ Future<void> getLostData() async {
     return;
   }
   if (response.file != null) {
-    setState(() {
-      if (response.type == RetrieveType.video) {
-        _handleVideo(response.file);
-      } else {
-        _handleImage(response.file);
-      }
-    });
+    if (response.files.length >= 2 ) {
+      _imageFileList = response.files
+    } else {
+      setState(() {
+        if (response.type == RetrieveType.video) {
+          _handleVideo(response.file);
+        } else {
+          _handleImage(response.file);
+        }
+    }});
```

Comment:
This example looks a bit confusing, as when dealing with multiple files it simply stores the list in the `_imageFileList` variable, while when dealing with a single file it is calling the `_handleVideo()` or `_handleImage()` method and updating the state (as per `setState` call). Seems these situations are handles very differently. Maybe you can update the code to a more simple example:

```dart
    if (response.files != null) {
      for(final XFile file in response.files) {
        _handleFile(file);  
      }
    } else {
      _handleError(response.exception);
    }
```

The above example simply shows how to handle lost data and leaves it to the implementing users to decide how to handle the files that are retrieved.

Reply:
@mvanbeusekom 
Thank you so much for your very thoughtful comments!
The example you wrote was very clear.
I've taken it in as a commit!